[docusaurus] Add to KB github docs

wallyrb opened the pull request:
Signed-off-by: Wally Rodriguez B <wallyrb@fb.com>

## Summary

1. Add guide to explain how to analyze service crashes
2. Add guide to explain how to override values in Orc8r deploy
3. Add guide to explain how to store terraform state files in S3
4. Add a guide to build/install a magma package
5. Add FAQ in how to flush redis

## Test Plan

Tested Locally
![image](https://user-images.githubusercontent.com/37117037/113937380-11ed9100-97c7-11eb-9358-4c9d3a3ed62e.png)
![image](https://user-images.githubusercontent.com/37117037/113937426-2598f780-97c7-11eb-8d2e-f7d946e73e64.png)
![image](https://user-images.githubusercontent.com/37117037/113937466-321d5000-97c7-11eb-822c-f6c81fe4a19f.png)
![image](https://user-images.githubusercontent.com/37117037/113937510-42cdc600-97c7-11eb-9785-b04d12d0036e.png)
![image](https://user-images.githubusercontent.com/37117037/113937528-4bbe9780-97c7-11eb-8834-ccdb57b62a9b.png)

wallyrb committed:
Add KB to github docs

Signed-off-by: Wally Rodriguez B <wallyrb@fb.com>

wallyrb commented on docs/readmes/orc8r/hold_terraform_state_on_s3.md:
```diff
@@ -0,0 +1,100 @@
+title: Hold Terraform state outside my local machine
+hide_title: true
+---
+# Holding the terraform state outside of my local machine
```
moving this KB to github so it can be visible to open source community. FYI  @sudhikan since you helped preparing this doc.

wallyrb commented on docs/readmes/orc8r/override_values_of_terraform_files.md:
```diff
@@ -0,0 +1,62 @@
+title: Override values of Terraform files
+hide_title: true
+---
+# Override values of Terraform files
```
moving this KB to github so it can be visible to open source community. FYI @sudhikan since you helped preparing this doc.

sudhikan commented on docs/readmes/orc8r/override_values_of_terraform_files.md:
```diff
@@ -0,0 +1,62 @@
+---
+# Override values of Terraform files
+
+**Description:** This guide drescribed how to override values that are part of the terraform files that are used in the github repository.
```
Can we change this to `This guide describes how to override values ....`

wallyrb replied:
Thank you! I have updated

andreilee commented on docs/readmes/faq/faq_magma.md:
```diff
@@ -79,6 +79,9 @@ This section lists some of the commonly asked questions related to Magma operati
   - **NOTE** Expect impact of service while doing above procedure.
 
+### Whats the command to flush the redis state?
+  - The command in AGW`redis-cli -p 6380 FLUSHALL` will flush the redis state
```
```suggestion
### What's the command to flush the redis state?
  - The command in AGW `redis-cli -p 6380 FLUSHALL` will flush the redis state
```

andreilee commented on docs/readmes/orc8r/hold_terraform_state_on_s3.md:
```diff
@@ -0,0 +1,100 @@
+    state_backend = "s3"
+    state_config = {
+        bucket = "s3_bucket_name"
+        region = "us-west-2"
```
I think it would be good to just link to one of our template terraform files and only include the relevant config here

andreilee replied:
Otherwise it may be tricky to keep this in sync with the template terraform file

wallyrb replied:
thanks for the feedback @andreilee ! comments have been addressed.

andreilee reviewed (changes requested):
Thanks for adding these. These are great. Can you add links/references to these docs in `deploy_install.md`? It's the `Install Orchestrator` guide, and has a lot of steps based on using terraform and kubernetes, and these docs are perfect supplementary material.

wallyrb commented:
> Thanks for adding these. These are great. Can you add links/references to these docs in `deploy_install.md`? It's the `Install Orchestrator` guide, and has a lot of steps based on using terraform and kubernetes, and these docs are perfect supplementary material.

Thanks a lot for the feedback. I have addressed the comments, feel free to suggest any other comments. Thanks!
